### ISSUE TITLE
Problem: The UI doesn't retain customizations by user from prior logins.

### ISSUE THREAD

**Issue by Robbt** [LibreTime/libretime#69]
Often times, navigating the track listing requires modifying the columns displayed and sort order. This is retained per session but whenever a user logs in again they need to set this up again. In practice this can be very time consuming and annoying, the alternative I'm proposing is for these settings to be stored per user and thus maintained between logins. This is the first step in improving the usability of the track UI.

Basically the column and sort order of the track UI will need to be saved somewhere in the database besides the session cookie.

**Comment by hairmare** [comment]
I think `Application_Model_Preferences` (and thus the cc_prefs table) is the right place to store these.

**Comment by hairmare** [comment]
The settings are already getting stored via a call to `/usersettings/set-library-datatable`. It looks like the gui isn't pulling them back into the sessing with `/usersettings/get-library-datatable`.
